module.json

```diff
@@ -1,21 +1,31 @@
 {
-    "name": "pf1-mighty-morphin",
+    "id": "pf1-mighty-morphin",
     "title": "PF1 Might Morphin Mod",
     "description": "Mod to automate polymorph effects for Foundry VTT PF1e system",
     "version": "0.3.0",
     "manifestPlusVersion": "1.0.0",
-    "minimumCoreVersion": "9",
-    "compatibleCoreVersion": "9",
+    "minimumCoreVersion": 10,
+    "compatibleCoreVersion": 10,
```

Comment:
These are obsolete since other v9 fields have been removed.

Reply:
I know. But v9 can't read the new syntax and thus will try to install the new module version, breaking everything for users (had that with my own module at first). So I'd suggest to leave it in and removeitsometime down the line when v9 has mostly phased out.